### PR TITLE
Clarify IP and hostname instructions

### PR DESCRIPTION
**Pull request by mattj-io**
Small fix to clarify README

### CODE REVIEW

File: tweeter/1.9/README.md [hunk 1]
```diff
@@ -54,9 +54,7 @@ In this step you deploy the containerized Tweeter app to a public node.
     $ git clone git@github.com:mesosphere/tweeter.git
     ```
 
-2.  Add the `HAPROXY_0_VHOST` label to the `tweeter.json` Marathon app definition file. `HAPROXY_0_VHOST` exposes Nginx on the external load balancer with a virtual host. The `HAPROXY_0_VHOST` value is the hostname of your [public agent][9] node.
-
-    **Important:** You must remove the leading `http://` and the trailing `/`.
+2.  Add the `HAPROXY_0_VHOST` label to the `tweeter.json` Marathon app definition file. `HAPROXY_0_VHOST` exposes Nginx on the external load balancer with a virtual host. The `HAPROXY_0_VHOST` value is the IP or hostname of your [public agent][9] node.
```
Comment (joerg84) [inline tweeter/1.9/README.md]: Or the ELB as you explain below.

File: tweeter/1.9/README.md [hunk 2]
```diff
@@ -92,7 +94,7 @@ In this step you deploy the containerized Tweeter app to a public node.
 
     The service talks to Cassandra via `node-0.cassandra.mesos:9042`, and Kafka via `broker-0.kafka.mesos:9557` in this example. Traffic is routed via the Marathon-LB (Marathon-LB) because you added the HAPROXY_0_VHOST tag on the `tweeter.json` definition.
 
-1.  Go to the DC/OS web interface to verify your app is up and healthy. Then, navigate to [public agent][9] node to see the Tweeter UI and post a Tweet.
+1.  Go to the DC/OS web interface to verify your app is up and healthy. Then, navigate to the [public agent][9] node to see the Tweeter UI and post a Tweet.
```
Comment (joerg84) [inline tweeter/1.9/README.md]: if you entered the ELB you need to use the ELB address

**Comment by mattj-io** [comment]
@joerg84 - changes as suggested

File: tweeter/1.9/README.md (hunk 2 above)
Comment (joerg84) [reply tweeter/1.9/README.md]: Then, navigate to the [public agent][9] node or the ELB address, depending on your configuration in step 2,  to see the Tweeter UI and post a Tweet.

**Comment by mattj-io** [comment]
@joerg84 changes as requested

**Pull request merged**